demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -50,6 +54,18 @@ def get_supported_native_images(
             return ScriptIntegrationSupportedNativeImages(
                 _id=self.object_id,
                 docker_image=self.docker_image,
-                native_image_config=file_to_native_image_config(),
+                native_image_config=NativeImageConfig.get_instance(),
             ).get_supported_native_image_versions(get_raw_version=True)
         return []
+
+    def get_python_version(self) -> Optional[Version]:
```

Comment:
is there no way to make a lazyloadproperty decorator to make this more generic?

Reply:
pyndatic do not support it at all unfortunately https://github.com/pydantic/pydantic/issues/1035

Reply:
Anyway I think this should be a property

Reply:
it might look more elegant as a property, but every time we create an integration script object it will be calculated and sometimes we don't need it (e.g. prepare content). I understood it adds 30 seconds to the graph calculation, so I'm in favor of avoiding these calculations where it's not needed.